identity/app/views/consentJourneyFragments/jsFallback.scala.html

```diff
@@ -30,7 +30,7 @@
 <noscript>
     @spinnerError(
         "noscriptError",
-        Html("Setting your communication preferences requires a browser with Javascript enabled. Please consider upgrading to one of our <a class=\"u-underline\" href=\"@LinkTo{/help/recommended-browsers}\">recommended browsers</a>. If you need more help please <a href=\"@LinkTo{/info/tech-feedback}\" class=\"u-underline\">contact us</a>."),
+        Html("Setting your communication preferences requires a browser with Javascript enabled. Please consider upgrading to one of our <a class=\"u-underline\" href=\"https://manage.theguardian.com/contact-us\">recommended browsers</a>. If you need more help please <a href=\"@LinkTo{/info/tech-feedback}\" class=\"u-underline\">contact us</a>."),
```

Comment:
Have you changed the wrong link in this paragraph? I don't think the recommended browser link should be changed.

Reply:
I did indeed. Thanks for catching.